tests/phpunit/tests/html-api/wpHtmlProcessor.php

```diff
@@ -331,7 +331,6 @@ public static function data_void_tags() {
 			'AREA'   => array( 'AREA' ),
 			'BASE'   => array( 'BASE' ),
 			'BR'     => array( 'BR' ),
-			'COL'    => array( 'COL' ),
```

Comment:
was this a mistake? I see no reason to treat `COL` as if it's not a void tag.

Reply:
No, this tag didn't work in the void tag tests because it's only present in tables. That was fixed as part of #6972 so I'll revert the change here.

Reply:
tests have been tricky at places, because we no longer have the concept of being able to step into _any_ tag, since stepping into that tag may ignore it or create other tags along the way 🙃